.coveragerc

```diff
@@ -48,6 +48,7 @@ omit =
     homeassistant/components/downloader.py
     homeassistant/components/keyboard.py
     homeassistant/components/light/hue.py
+    homeassistant/components/light/mqtt.py
```

Comment:
That being said, I don't expect extensive tests, but at least a sanity test to ensure that the state topics work correctly for example would be nice.